docs/0_internal/5_b_1_propose_whitelist.md

```diff
@@ -0,0 +1,59 @@
+# Propose the Transaction for Whitelisting Operators using Multisig
+
+If you want to Whitelist Operators, you can propose the whitelist operator transaction using the multisig wallet.
+
+## Prerequisites
+
+- You need to have deployed the contracts following the [Deploy Contracts Guide](./2_deploy_contracts.md).
+
+## Propose transaction for Whitelist Operators
+
+To propose the whitelist transaction you can follow the steps below:
+
+1. Go to [Safe](https://app.safe.global/home)
+
+2. Click on `New transaction` -> `Transaction Builder`
+
+   ![New transaction](./images/5_b_1_whitelist_operator_1.png)
+
+   ![Transaction Builder](./images/5_b_1_whitelist_operator_2.png)
+
+3. Get the `registryCoordinator` address from ```contracts/script/output/mainnet/alignedlayer_deployment_output.json``` or ```contracts/script/output/holesky/alignedlayer_deployment_output.json``` or ```contracts/script/output/sepolia/alignedlayer_deployment_output.json```
```

Comment:
```suggestion
3. Get the `RegistryCoordinator` address from ```contracts/script/output/mainnet/alignedlayer_deployment_output.json``` or ```contracts/script/output/holesky/alignedlayer_deployment_output.json``` or ```contracts/script/output/sepolia/alignedlayer_deployment_output.json```
```

Reply:
In the contract deployment config `registryCoordinator` is not capitalized. I think making it true to the config would help dissuade confusion.
![Uploading Screenshot 2024-11-29 at 14.23.52.png…]()